packages/acot-preset-wcag/docs/rules/img-has-name.md

```diff
@@ -0,0 +1,28 @@
+# img-has-name
+
+> img MUST has name
```

Comment:
Please use the short summary of rules in paragraphs :pray:

Reply:
[nits]

```suggestion
> img element MUST has name.
```

Reply:
thanks,  I will add `.` and `element or img role`